src/tests/components/App.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import App from '../../components/App';
+
+describe('<App />', () => {
+  it('renders three <App /> components', () => {
```

Comment:
Good job @nwashangai. Please, can you make the describe block more descriptive so we can know what the test suite is all about.